tests/lib/outputs/html/html_test.py

```diff
@@ -0,0 +1,526 @@
+                </div>
+                <div class="col-md-4">
+                <div class="card">
+                    <div class="card-header">
+                        Azure Credentials
+                    </div>
+                    <ul class="list-group list-group-flush">
+                        <li class="list-group-item">
+                            <b>Azure Identity Type:</b> Service Principal
+                            </li>
+                            <li class="list-group-item">
+                                <b>Azure Identity ID:</b> 00000000-0000-0000-0000-000000000000
+                            </li>
+                        </ul>
+                    </div>
+                </div>"""
+
+gcp_html_assessment_summary = """
+                <div class="col-md-2">
+                    <div class="card">
+                        <div class="card-header">
+                            GCP Assessment Summary
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            <li class="list-group-item">
+                                <b>GCP Project IDs:</b> 123456789012
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-md-4">
+                    <div class="card">
+                        <div class="card-header">
+                            GCP Credentials
+                        </div>
+                        <ul class="list-group list-group-flush">
+                            <li class="list-group-item">
+                                <b>GCP Account:</b> test@test.com
+                            </li>
+                        </ul>
+                    </div>
+                </div>"""
+
+kubernetes_html_assessment_summary = """
+                <div class="col-md-2">
+                    <div class="card">
+                        <div class="card-header">
+                            Kubernetes Assessment Summary
+                        </div>
+                        <ul class="list-group
+                        list-group-flush">
+                            <li class="list-group-item">
+                                <b>Kubernetes Cluster:</b> None
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="col-md-4">
+                    <div class="card">
+                        <div class="card-header">
+                            Kubernetes Credentials
+                        </div>
+                        <ul class="list-group
+                        list-group-flush">
+                            <li class="list-group-item">
+                                <b>Kubernetes Context:</b> None
+                            </li>
+                        </ul>
+                    </div>
+                </div>"""
+
+
+@pytest.fixture
+def generate_finding():
```

Comment:
from tests.lib.outputs.fixtures.fixtures you could use generate_finding_output, wait until #4368 it's merged

Reply:
This PR have several changes for this method and we could remove this fixture